modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -436,6 +436,8 @@ def hlr_create_schemas
 
   def decision_reviews_hlr_response_schemas
     schemas = hlr_response_schemas
+    schemas = deep_replace_key(schemas, :createDate, :createdAt)
+    schemas = deep_replace_key(schemas, :updateDate, :updatedAt)
```

Comment:
I spent a while trying to refactor this file so that we didn't need to reuse and then edit these schemas, but the number of lines changed would have reached almost 3000 - it didn't seem pressing enough to split out into several PRs right now, since we're moving this code to `lighthouse-vets-api` soon, so I wrote this helper to just rename the keys instead.

Reply:
I agree with your approach here.